machineconfiguration/v1/tests/controllerconfigs.machineconfiguration.openshift.io/AWSClusterHostedDNS.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Shamefully missing"
```

Comment:
Lol, lets make this sensible before we merge, or was this copy/pasted from an existing place?

Reply:
This was actually copied over directly from the GCP one.